servers.json

```diff
@@ -1429,5 +1429,12 @@
         "addresses": [
             "ymzc.pro"
         ]
+    {
```

Comment:
Please add a closing bracket for the previous entry.

Reply:
Okay I fixed it